x-pack/plugins/triggers_actions_ui/public/application/sections/rules_list/components/rule_tag_badge.tsx

```diff
@@ -16,55 +16,61 @@ const tagTitle = i18n.translate(
   }
 );
 
+export type RuleTagBadgeOptions = 'all' | 'default';
```

Comment:
I wanted to this update invisible (no update required after this change) for any component/page uses`RuleTag`. But does introduce the `default` option will require updating all component uses tags? 

nit: `all` is very generic and doesn't reflect the functionality.